DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/bridge/linux/linux_bridge_vlan_impl.py

```diff
@@ -18,10 +18,10 @@ def format_update(self, command, *argv, **kwarg):
         params = kwarg['params']
         cmd = 'bridge {} vlan {} '.format(params.get('cmd_options', ''), command)
         # custom code here
-        if 'device' in params:
-            cmd += 'dev {} '.format((params['device']))
         if 'vid' in params:
             cmd += 'vid {} '.format((params['vid']))
+        if 'device' in params:
+            cmd += 'dev {} '.format((params['device']))
```

Comment:
What is the intention of this change? bridge utility takes dev as the first parameter and vid as the second one.